blog/2025/pcsx2-2.6/index.mdx

```diff
@@ -0,0 +1,284 @@
+
+Following [#13614](https://github.com/PCSX2/pcsx2/pull/13614) it was noticed that NVIDIA lost the smooth scaling unlike AMD retained.
+This issue was identified as due to a difference in how AMD and NVIDIA reported resized windows. As a result, we would recreate the swapchain at the end of the frame on NVIDIA, but at the beginning of the frame on AMD.
+During resizing, the swapchain would become invalidated between the end of one frame and the beginning of the next, causing the rendering on NVIDIA to seemingly freeze.
+
+The fix was to simply defer recreating the swapchain until the beginning of the frame.
+
+#### But Wait, There's More!
+
+It was noticed that RDNA 4 would experience significant delay in resizing even after #13614 was merged.
+
+This was determined to due to a vulkan function `vkAcquireNextImageKHR` taking 2 seconds to complete following a swapchain re-creation. A workaround for this driver bug was to not link the old swapchain to the new one.
+
+While implementing this fix, support for `VK_KHR_swapchain_maintenance1` was added. This may reduce the memory usage from resizing on AMD GPUs with updated drivers. NVIDIA supported the older version we had already been using, so no change is expected there.
+
+## Various Smaller Improvements in 2.6
+
+<PCSX2PRLink authors="kamfretoz,SternXD" prNums="12848,12945">
+  ### Automatically Launch Big Picture Mode on Startup
+</PCSX2PRLink>
+
+For those who wants the true living room couch gaming experience, you can now launch PCSX2 directly into Big Picture Mode and starts gaming right away!
+
+<PCSX2PRLink authors="TheLastRar,F0bes" prNums="13403,13592">
+  ### Improved Mouse Lock DPI Handling
+</PCSX2PRLink>
+
+This improves upon ([#13403](https://github.com/PCSX2/pcsx2/pull/13403)) while also fixing issues introduced in the QWindow update. Mainly this fixes mouse lock on setups with multi monitors using DPI > 100%.
+
+Now mouse lock should function correctly regardless of your monitor setup on Windows and Mac.
+
+<PCSX2PRLink authors="TheLastRar" prNums="13623">
+  ### Support Full Axis Binds of Gamepad Axes
+</PCSX2PRLink>
+
+Only some axes could be bound as full axis with SDL.
+This isn't normally needed, so this had been missed.
+Now this has been fixed.
+
+<PCSX2PRLink authors="TheTechnician27" prNums="13015">
+  ### Improved Hotkeys Section
+</PCSX2PRLink>
+
+TheTechnician27 worked on rearranging and reorganizing the hotkey section to give you the more cohesive user experience.
+
+<Image src={require("./img/hotkey.webp").default} />
+
+<PCSX2PRLink authors="TheLastRar" prNums="12970,12837">
+  ### Big Picture Mode Icon Refresh
+</PCSX2PRLink>
+
+Big Picture Mode just got a small facelift! You might notice the inconsistencies that some options have an icon while other does not. Now every settings entry now has icon to go along with it.
```

Comment:
"You might have noticed inconsistencies where some options had an icon while others did not. Now, every settings entry has icon to go along with it."